projects/aws/eks-anywhere-build-tooling/build/organize_binaries.sh

```diff
@@ -39,7 +39,8 @@ declare -A project_bin_licenses=(["eksa/fluxcd/flux2"]="flux FLUX2"
                                  ["eksa/replicatedhq/troubleshoot"]="support-bundle TROUBLESHOOT"
                                  ["eksa/vmware/govmomi"]="govc GOVMOMI"
                                  ['eksa/helm/helm']="helm HELM"
-                                 ['eksa/tinkerbell/tink']="tink TINK")
+                                 ['eksa/tinkerbell/tink']="tink TINK"
+                                 ["eksa/apache/cloudstack-cloudmonkey"]="cmk CLOUDMONKEY")
```

Comment:
Totally right way to do this! Calling this out as a note that we should eventually rethink our cli tools strategy so we dont end up with a giant image with every provider in it.

Reply:
agreed, it may make sense to have separate CLI tool images per usecase, and then maybe a set of long running docker containers